db/migrate/20250814153656_add_compound_primary_key_to_mhv_metrics_unique_user_events.rb

```diff
@@ -0,0 +1,14 @@
+class AddCompoundPrimaryKeyToMHVMetricsUniqueUserEvents < ActiveRecord::Migration[7.2]
+  disable_ddl_transaction!
```

Comment:
From what I'm [reading about `disable_ddl_transaction!`](https://medium.com/@alie.ammaar/understanding-disable-ddl-transaction-in-rails-migrations-a-comprehensive-guide-b5415295708d), you should only need this when performing migrations on very large tables that already exist. In this case, the migration before it created a brand new table.

Reply:
I'm reading that our [Platform Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-database-migrations#vets-apidatabasemigrations-IndexAddition) recommend your approach, so feel free to ignore this comment.